app/services/site_channel_host_inspector.rb

```diff
@@ -30,13 +30,11 @@ def inspect_uri(uri)
                    "github"
                  elsif response.body.include?("/wp-content/")
                    "wordpress"
-                 else
-                   nil
                  end
     end
 
     { response: response, web_host: web_host }
-  rescue Publishers::Fetch::RedirectError, Publishers::Fetch::ConnectionFailedError => e
+  rescue => e
```

Comment:
Wouldn't line 24 throw an exception and prevent us from knowing some SSL errors?

Reply:
Yep, the `fetch` will raise an exception but since this is caught the exception is returned in the response it's able to be set in the `response_result`